From 1b6bb07fa4eedbedf85f413ccaa070aa8f659fec Mon Sep 17 00:00:00 2001
From: Chris Mayo <aklhfex@gmail.com>
Date: Tue, 7 Jun 2016 20:01:55 +0100
Subject: [PATCH 3/7] remove desktop Version entry

Version is the version of the Desktop Entry Specification not Fritzing

(cherry picked from commit 789149760451defdf55a27647fcf1c20105cb2f8)
---
 fritzing.desktop | 1 -
 1 file changed, 1 deletion(-)

diff --git a/fritzing.desktop b/fritzing.desktop
index 5b14537..6b88de2 100644
--- a/fritzing.desktop
+++ b/fritzing.desktop
@@ -1,5 +1,4 @@
 [Desktop Entry]
-Version=0.9.3b
 Name=Fritzing
 GenericName=Fritzing
 Comment=Electronic Design Automation software
-- 
2.8.3

